.github/workflows/snarkjs.sh

```diff
@@ -9,6 +9,10 @@ fi
 DIR_PATH=$1
 CURVE=$2
 
+# Init stdlib in .noname/release/src/stdlib instead of downloading
+echo "Overriding stdlib in .noname/release/src/stdlib..."
+mkdir -p ~/.noname/release/src/stdlib/ && cp -r /app/noname/src/stdlib/* ~/.noname/release/src/stdlib/
```

Comment:
I don't think we should do this manually BTW, we should let noname do it

Reply:
This is because the CI test wants to test against the latest code, including the stdlib. To ensure the latest code, here overrides the default downloaded version with the code from the PR.